src/OmniSharp/Models/NavigateResponse.cs

```diff
@@ -0,0 +1,9 @@
+namespace OmniSharp.Models
+{
+    public class NavigateResponse
+    {
+        public string FileName { get; set; }
```

Comment:
will you ever end up in a different file? Or should this be a a _code pointer_ thing (line and position)? Iff so it should have a different name